### PR TITLE
Bump main IPs

### PR DESCRIPTION
**Pull request by alex96295**
- [x] **Safety island:** input sync, triple-core lockstep (TCLS), bus_err_unit @micprog 
- [x] **Security island:** expose SCK and CSB output enable for SPI @maicolciani 
- [x] **Integer cluster:** re-point to existing commit after force push (do not
force push :smiley: ) @yvantor 
- [x] Init carfield with `bender checkout` from the Lock file instead of `bender update`

### CODE REVIEW

File: Bender.yml [hunk 1]
```diff
@@ -13,9 +13,9 @@ dependencies:
...
   hyperbus:           { git: https://github.com/pulp-platform/hyperbus.git,             rev: 80f137ce8904c78602b589918911f388d507e935 }
   car_l2:             { git: git@iis-git.ee.ethz.ch:carfield/carfield_l2_mem.git,       rev: 505c35a8df7f042fc8fc07c5dc40bc6412f7b27d } # branch: main
-  safety_island:      { git: git@iis-git.ee.ethz.ch:carfield/safety-island.git,         rev: 0c5c15dac99a5cf554102d1c579a8c9c768b5207 } # branch: master
-  pulp_cluster:       { git: https://github.com/pulp-platform/pulp_cluster.git,         rev: 2376e7d79e77a32863ff845275857a8694f7e225 } # branch: yt/carfield-integration
-  opentitan:          { git: https://github.com/alsaqr-platform/opentitan.git,          rev: c4a1b1d9bca7ca443a64b9f7f5586afa7a28cc5f } # branch: lowRISC-rebase
+  safety_island:      { git: git@iis-git.ee.ethz.ch:carfield/safety-island.git,         rev: c03f1d0b7aad5725010f247130e5d13b36c89c04 } # branch: master
+  pulp_cluster:       { git: https://github.com/pulp-platform/pulp_cluster.git,         rev: f18f07d7c99dd3d97197636610112b359c14af3c } # branch: yt/carfield-integration
```
Comment (yvantor) [inline Bender.yml:17]: I suggest to update the commit to [the recent one](https://github.com/pulp-platform/pulp_cluster/commit/91ec9e6539e6e6a118756eeba73cb5013b15288f)

**Comment by alex96295** [comment]
Rebased onto main

File: Bender.yml (hunk 1 above)
Comment (alex96295) [reply Bender.yml]: It's done now, it needed rebase onto main

**Comment by yvantor** [comment]
lgtm

**Pull request merged**